modules/serverless-logic-upgrade-1-36-deleting-migrating-workflows-with-preview-profile.adoc

```diff
@@ -0,0 +1,22 @@
+// Module included in the following assemblies:
+// * serverless-logic/serverless-logic-upgrading-operator-from-1.35-to-1.36
+
+
+:_mod-docs-content-type: PROCEDURE
+[id="serverless-logic-upgrade-1-36-deleting-migrating-workflows-with-preview-profile_{context}"]
+= Deleting workflows with the preview profile
+
+Before upgrading the Operator, you must delete workflows running with the `preview` profile. When the upgrade is complete, you must redeploy the workflows.
+
+.Procedure
+
+. If you are using persistence, back up the workflow database and ensure the backup includes both database objects and table data.
+
+. Ensure you have a backup of all necessary Kubernetes resources, including `SonataFlow` custom resources (CRs), `ConfigMaps`, or any other related custom configurations.
+
+. Delete the workflow by executing the following command: 
++
+[source,terminal]
+----
+$ oc delete -f <workflow_name>.yaml -n <target_namespace>
```

Comment:
```suggestion
$ oc delete -f <workflow_name> -n <target_namespace>
```

Don't need to add the file extension.

Reply:
In this case, I think we must remove the -f  and type  `oc delete workflow <workflow_name> -n <target_namespace> `

@kaldesai   @ricardozanini